I, [2018-03-01T11:01:47.256702 #16100]  INFO -- Kitchen: -----> Starting Kitchen (v1.19.2)
I, [2018-03-01T11:01:53.213352 #16100]  INFO -- Kitchen: -----> Creating <default-centos-72>...
E, [2018-03-01T11:01:57.619394 #16100] ERROR -- Kitchen: ------Exception-------
E, [2018-03-01T11:01:57.619394 #16100] ERROR -- Kitchen: Class: Kitchen::ActionFailed
E, [2018-03-01T11:01:57.619394 #16100] ERROR -- Kitchen: Message: 1 actions failed.
>>>>>>     Failed to complete #create action: [Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider virtualbox ----
STDOUT: Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'bento/centos-7.2' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
STDERR: The box 'bento/centos-7.2' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/bento/centos-7.2"]
Error: The requested URL returned error: 404 Not Found
---- End output of vagrant up --no-provision --provider virtualbox ----
Ran vagrant up --no-provision --provider virtualbox returned 1] on default-centos-72
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: ----------------------
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: ------Backtrace-------
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:183:in `report_errors'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:174:in `run_action'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command/action.rb:36:in `block in call'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command/action.rb:34:in `call'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/cli.rb:52:in `perform'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/cli.rb:193:in `block (2 levels) in <class:CLI>'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
E, [2018-03-01T11:01:57.620395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/bin/kitchen:13:in `block in <top (required)>'
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/errors.rb:171:in `with_friendly_errors'
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/bin/kitchen:13:in `<top (required)>'
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/bin/kitchen:267:in `load'
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/bin/kitchen:267:in `<main>'
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: ----End Backtrace-----
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: -Composite Exception--
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: Class: Kitchen::ActionFailed
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: Message: Failed to complete #create action: [Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider virtualbox ----
STDOUT: Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'bento/centos-7.2' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
STDERR: The box 'bento/centos-7.2' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/bento/centos-7.2"]
Error: The requested URL returned error: 404 Not Found
---- End output of vagrant up --no-provision --provider virtualbox ----
Ran vagrant up --no-provision --provider virtualbox returned 1] on default-centos-72
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: ----------------------
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: ------Backtrace-------
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/shell_out.rb:71:in `rescue in run_command'
E, [2018-03-01T11:01:57.621395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/shell_out.rb:59:in `run_command'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:435:in `run_command'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:382:in `run'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:466:in `run_vagrant_up'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:107:in `create'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:469:in `public_send'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:469:in `block in perform_action'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:533:in `block in synchronize_or_call'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:531:in `synchronize'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:531:in `synchronize_or_call'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:498:in `block in action'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:497:in `action'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:469:in `perform_action'
E, [2018-03-01T11:01:57.622394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:379:in `create_action'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:368:in `block in transition_to'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `each'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `transition_to'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:124:in `create'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `public_send'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: ----End Backtrace-----
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: ---Nested Exception---
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: Class: Kitchen::ActionFailed
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: Message: Failed to complete #create action: [Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider virtualbox ----
STDOUT: Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'bento/centos-7.2' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
STDERR: The box 'bento/centos-7.2' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/bento/centos-7.2"]
Error: The requested URL returned error: 404 Not Found
---- End output of vagrant up --no-provision --provider virtualbox ----
Ran vagrant up --no-provision --provider virtualbox returned 1]
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: ----------------------
E, [2018-03-01T11:01:57.623395 #16100] ERROR -- Kitchen: ------Backtrace-------
E, [2018-03-01T11:01:57.624394 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/shell_out.rb:71:in `rescue in run_command'
E, [2018-03-01T11:01:57.627412 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/shell_out.rb:59:in `run_command'
E, [2018-03-01T11:01:57.627901 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:435:in `run_command'
E, [2018-03-01T11:01:57.627901 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:382:in `run'
E, [2018-03-01T11:01:57.627901 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:466:in `run_vagrant_up'
E, [2018-03-01T11:01:57.627901 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-vagrant-1.2.1/lib/kitchen/driver/vagrant.rb:107:in `create'
E, [2018-03-01T11:01:57.627901 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:469:in `public_send'
E, [2018-03-01T11:01:57.627901 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:469:in `block in perform_action'
E, [2018-03-01T11:01:57.628402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:533:in `block in synchronize_or_call'
E, [2018-03-01T11:01:57.628402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:531:in `synchronize'
E, [2018-03-01T11:01:57.628402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:531:in `synchronize_or_call'
E, [2018-03-01T11:01:57.628402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:498:in `block in action'
E, [2018-03-01T11:01:57.628402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2018-03-01T11:01:57.628902 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:497:in `action'
E, [2018-03-01T11:01:57.628902 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:469:in `perform_action'
E, [2018-03-01T11:01:57.628902 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:379:in `create_action'
E, [2018-03-01T11:01:57.628902 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:368:in `block in transition_to'
E, [2018-03-01T11:01:57.628902 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `each'
E, [2018-03-01T11:01:57.628902 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:367:in `transition_to'
E, [2018-03-01T11:01:57.629402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/instance.rb:124:in `create'
E, [2018-03-01T11:01:57.629402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `public_send'
E, [2018-03-01T11:01:57.629402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2018-03-01T11:01:57.629402 #16100] ERROR -- Kitchen: D:/opscode/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.19.2/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2018-03-01T11:01:57.629402 #16100] ERROR -- Kitchen: ----End Backtrace-----
